### PR TITLE
Make loading prop available to child components

### PR DESCRIPTION
**Pull request by jruhland**
* Bump to version 7.2.0
  * Add loading to internal state
  * Expose loading status to render prop
  * Add hooks for modifying loading status

Exposing loading status to render so that users can easily add spinners/etc during loading

### CODE REVIEW

**Comment by jruhland** [comment]
@kenny-hibino any feedback on this?

**Comment by kenny-hibino** [comment]
Hey @jruhland Thanks for submitting this PR!
I really like this, let me test it out and merge it sometime this week 👍 
In the meantime, would you mind adding a simple test for this if you can?

**Comment by jruhland** [comment]
I'd be happy to

**Comment by jruhland** [comment]
Wasn't 100% sure what pattern to follow in the repository, so this is best effort, if this isn't what you had in mind, please let me know.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-places-autocomplete",
-  "version": "7.1.2",
+  "version": "7.2.0",
```
Comment (hibiken) [inline package.json]: I will bump the version when I push this to npm registry, so no need to make this change in this PR 👍

Comment (jruhland) [reply package.json]: addressed in https://github.com/hibiken/react-places-autocomplete/pull/214/commits/f25dcc7447476c849723717fe2bb56d9c34eac03

**Comment by jruhland** [comment]
Addressed the requested changes, and squashed commits for a nicer history.  ❤️thanks for being an awesome maintainer!

**Pull request merged**